### ISSUE TITLE
Clarify licensing for sexual and romantic identities

### ISSUE THREAD

**Issue by bradjones1** [drnikki/open-demographics#72]
[This page](https://github.com/drnikki/open-demographics/blob/master/doc-source/questions/sexualorientation.rst) says these identity taxonomies are "taken from" a book, but there's no specific information about this data being licensed compatible with the creative commons license applied to the broader dataset. Can this be clarified? This list isn't particularly revolutionary, but it would be nice to ensure proper licensing if it's not original work to this project. Thanks.

**Comment by germonprez** [comment]
Thanks @bradjones1. I'm not sure where these were sourced from. Your point is well taken to provide proper attribution.

**Comment by GeorgLink** [comment]
The page does provide attribution to the book where the list comes from. Is there a "license" required beyond attribution?

**Comment by bradjones1** [comment]
> The page does provide attribution to the book where the list comes from. Is there a "license" required beyond attribution?

Definitely. The book is copyrighted work and providing attribution doesn't change anything about whether it's licensed for use in other works. Everything we write is copyrighted, but licensing is what makes the re-use and rework possible. For context, see https://creativecommons.org/share-your-work/licensing-considerations/compatible-licenses/ and similar. Not trying to make a mountain out of a molehill, but since this project is licensed requiring attribution and that content is taken from another source with no licensing called out, I'm curious if the author provided permission.

**Comment by GeorgLink** [comment]
In academia it's common practice to reference other works without the need to obtain a license, so long as we don't re-print substantial parts of an original works. 

Here in question is a list of identities. This can be found in other places. I believe that pointing to a book that did a great job of describing the identities is sufficient. 

Maybe we can do a better job of referencing the book with something like this:

> For more information about sexual and romantic identities we recommend the book "The ABCs of LGBTQ+" by Ashley Mardell. Available at https://mango.bz/books/the-abcs-of-lgbt-by-ashley-mardell-113-b

(I'm not a lawyer)
